### PR TITLE
feat: preview domains

### PR DESCRIPTION
**Pull request by omBratteng**
😎

### Preview domain
https://as-203.preview.app.daily.dev

### CODE REVIEW

File: .github/workflows/add-preview-domain.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+name: Assign preview-domain
+on:
+  # Trigger when a pull request is opened or reopened
+  pull_request:
+    types: [opened, reopened]
```
Comment (capJavert) [inline .github/workflows/add-preview-domain.yml:5]: Just as precaution, can we somehow limit to check if team member opened the PR? Someone could spam a lot of PRs and register many domains in our Vercel account.

File: .github/workflows/remove-preview-domain.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: Unassign preview-domain
+on:
+  # Trigger when a pull request is closed
+  pull_request:
+    types: [closed]
```
Comment (capJavert) [inline .github/workflows/remove-preview-domain.yml:5]: Nice 👌

**Review by capJavert — APPROVED** [review]
Love it!

File: .github/workflows/add-preview-domain.yml (hunk 1 above)
Comment (capJavert) [reply .github/workflows/add-preview-domain.yml:5]: Our plan has unlimited domains, but if it is easy to somehow check it would be great.

Comment (omBratteng) [reply .github/workflows/add-preview-domain.yml:5]: We can protect it with this setting
image

But I'm pretty sure I can whip something together to ensure the author is from our team/ github org

File: .github/workflows/remove-preview-domain.yml (hunk 2 above)
Comment (omBratteng) [reply .github/workflows/remove-preview-domain.yml:5]: Yup. A good DevOps cleans up after themselves too

File: .github/workflows/add-preview-domain.yml (hunk 1 above)
Comment (capJavert) [reply .github/workflows/add-preview-domain.yml:5]: Yes, the above would also work as a first measure 👌

Comment (omBratteng) [reply .github/workflows/add-preview-domain.yml:5]: Changed the setting :)

**Review by DragosIorgulescu — APPROVED** [review]
nice 💯

File: .github/workflows/add-preview-domain.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            const knownString = '### Preview domain'
+            const pullRequest = await github.rest.pulls.get({
+              pull_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            })
+            const hasAlreadyCommented = pullRequest.data.body.includes(knownString)
+
+            if (hasAlreadyCommented) {
+              console.log('Already commented once')
```
Comment (rebelchris) [inline .github/workflows/add-preview-domain.yml:37]: Do the consoles add anything?

**Review by rebelchris — APPROVED** [review]
Looks good to me

File: .github/workflows/add-preview-domain.yml (hunk 3 above)
Comment (omBratteng) [reply .github/workflows/add-preview-domain.yml:37]: Just for debugging mostly
https://github.com/dailydotdev/apps/actions/runs/8392439290/job/22985086956#step:3:34

**Review by kopancek — APPROVED** [review]
Nothing to add, looks good to me. Cannot wait 🤩

**Pull request merged**